tests/unit/test_world_scientific.py

```diff
@@ -124,8 +124,8 @@ def test_abstract(generated_record, expected_abstract):
             [{
                 'source': 'WSP',
                 'title': (
-                    'High-efficient Solid-state Perovskite Solar Cell Without '
-                    'Lithium Salt in the Hole Transport Material'
+                    'HIGH-EFFICIENT SOLID-STATE PEROVSKITE SOLAR CELL WITHOUT '
+                    'LITHIUM SALT IN THE HOLE TRANSPORT MATERIAL'
```

Comment:
What happened here? Why the caps change?

Reply:
https://github.com/inspirehep/hepcrawl/pull/180/commits/490888a6760f03dbefcc0d7eefdbae9a098fead3 happened.

Reply:
Oh... so it was like that in the original record :/ okok